rpm/firejail.spec

```diff
@@ -64,7 +65,6 @@ rm -rf %{buildroot}%{_datadir}/zsh/site-functions/_firejail
 %files profiles
 %{_bindir}/firecfg
 %{_bindir}/jailcheck
-%{_libdir}/%{name}/firecfg.config
```

Comment:
Please also drop the corresponding exclude above.
```
%exclude %{_libdir}/%{name}/firecfg.config
```

Reply:
Actually, if they install firecfg.config to /etc, we probably should have it in this subpackage instead. In that case change this and the `%exclude` to use `%{_sysconfdir}` instead of `%{_libdir}`.